tools.json

```diff
@@ -576,6 +576,18 @@
         }
       },
       "test": "testability --help"
+    },
+    {
+      "name": "CoverageChecker",
+      "summary": "Applies QA tools to run on a single pull request",
+      "website": "https://github.com/exussum12/coverageChecker",
+      "command": {
+        "composer-bin-plugin": {
+          "package": "exussum12/coverage-checker:dev-master",
```

Comment:
Can we use `exussum12/coverage-checker` here so that the latest stable version is installed?

Reply:
Changed, I just copied the one from above so assumed that was the way to do it

Reply:
Have you pushed? I can't see your changes :)

Reply:
odd - I used the web editor. Will just use regular git

Reply:
https://github.com/jakzal/phpqa/compare/master...exussum12:patch-1  It does appear do wont let me push. I will take a look tonight. Thanks